core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -74,6 +74,13 @@
     FileInStream open(URIStatus uriStatus) throws IOException, AlluxioException;
   }
 
+  /**
+   * Registers metrics
```

Comment:
/usr/src/alluxio/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java:77: First sentence should end with a period. [JavadocStyle]